django/contrib/gis/measure.py

```diff
@@ -290,7 +291,8 @@ class Distance(MeasureBase):
         'US survey foot': 'survey_ft',
         'U.S. Foot': 'survey_ft',
         'Yard (Indian)': 'indian_yd',
-        'Yard (Sears)': 'sears_yd'
+        'Yard (Sears)': 'sears_yd',
+        'Furlongs': 'furlong'
```

Comment:
I think `Furrow long` should be an alias instead of `Furlongs`, also please alphabetize.